src/parser/mod.rs

```diff
@@ -3869,11 +3873,16 @@ impl<'a> Parser<'a> {
             .is_some();
         let persistent = dialect_of!(self is DuckDbDialect)
             && self.parse_one_of_keywords(&[Keyword::PERSISTENT]).is_some();
+        let create_view_params = if dialect_of!(self is MySqlDialect | GenericDialect) {
+            self.parse_create_view_params()?
+        } else {
+            None
+        };
```

Comment:
maybe this could be accepted without the dialect check if that works just as well `let create_view_paras = self.parse_create_view_params()`?  thinking since the view params are optional